code/obj/item/organs/head.dm

```diff
@@ -31,6 +31,7 @@
 	/// Defines what kind of head this is, for things like lizards being able to colorchange a transplanted lizardhead
```

Comment:
Also add something to `disposing()` that (1) checks for skeleton mutantrace and if so sets `head_tracker` to null (2) sets `linked_human` to null